src/function/unit/toBest.js

```diff
@@ -0,0 +1,46 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'toBest'
+const dependencies = ['typed']
+
+export const createToBest = /* #__PURE__ */ factory(name, dependencies, ({ typed }) => {
+  /**
+   * Converts a unit to the most appropriate display unit.
+   * When no preferred units are provided, the function automatically find the best prefix.
+   * When preferred units are provided, it converts to
+   * the unit that gives a value closest to 1.
+   *
+   * Syntax:
+   *
+   *    math.toBest()
+   *    math.toBest(unitList)
+   *    math.toBest(unitList, options)
+   *
+   * Where:
+   * - `unitList` is an optional array of preferred target units as string or Unit.
+   * - `options` is an optional object with options, formed as follows:
+   * - `offset`: number | BigNumber
+   *
+   * Examples:
```

Comment:
Can you change the example notation from:

```
math.unit(10, 'm').toBest(['cm', 'mm'])                           // returns Unit(500, 'cm')
math.unit(2 / 3, 'cm').toBest()                                   // returns Unit(0.6666666666666666, 'cm')
math.unit(10, 'm').toBest(['mm', 'km'], { offset: 1.5 })          // returns Unit(10000, 'mm')
```

to

```
math.unit(0.05, 'm').toBest(['cm', 'mm'])                         // Unit 500 cm
math.unit(2 / 3, 'cm').toBest()                                   // Unit 0.6666666666666666 cm
math.unit(10, 'm').toBest(['mm', 'km'], { offset: 1.5 })          // Unit 10000 mm
```

That will make the script that validates the examples in the comments happy 😊 .